src/main/java/Event.java

```diff
@@ -0,0 +1,29 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
```

Comment:
Felt that most of your description for the constructors should have been for the class.